src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,10 @@
-__version__ = "1.2.2"
+__version__ = "1.3.0.dev0"
```

Comment:
`versioneer` also provides a convenient command class to inject into setup.py to handle that kind of thing.